defaults.yaml

```diff
@@ -76,7 +76,19 @@ ansible:
       # Allowed values are: weavenet, calico, flannel, and no-cni-plugin.
       # If you choose no-cni-plugin, provisioners will prepare the environment
       # but will NOT bootstrap the Kubernetes cluster.
-      kubernetes_network_plugin: calico
+      kubernetes_network_plugin_options:
```

Comment:
Can you please update the comments for this parameters and comments for the ones you added?

Reply:
done!